examples/plot_model_card.py

```diff
@@ -64,45 +64,50 @@
 # Then, we pass information other than metadata in kwargs.
 # We'll initialize a local repository and save the card with the model in it.
 
+license = "mit"
+
+
+model_card = card.Card(model)
+
+
+_, pkl_name = mkstemp(prefix="skops-", suffix=".pkl")
+
+with open(pkl_name, mode="bw") as f:
+    pickle.dump(model, file=f)
+
+local_repo = mkdtemp(prefix="skops-")
+hub_utils.init(
+    model=pkl_name, requirements=[f"scikit-learn={sklearn.__version__}"], dst=local_repo
+)
+
 limitations = "This model is not ready to be used in production."
 model_description = (
     "This is a HistGradientBoostingClassifier model trained on breast cancer dataset."
     " It's trained with Halving Grid Search Cross Validation, with parameter grids on"
     " max_leaf_nodes and max_depth."
 )
-license = "mit"
-
-card_data = CardData(
-    license=license,
-    tags=["tabular-classification"],
-    datasets="breast-cancer",
-    metrics=["acc"],
-)
-
 model_card_authors = "skops_user"
 get_started_code = (
     "import pickle\nwith open(dtc_pkl_filename, 'rb') as file:\nclf = pickle.load(file)"
 )
-citation = "bibtex\n@inproceedings{...,year={2020}}"
-
-model_card = card.create_model_card(
-    model,
-    card_data=card_data,
-    limitations=limitations,
-    model_description=model_description,
-    citation_bibtex=citation,
-    model_card_authors=model_card_authors,
-    get_started_code=get_started_code,
+
+model_card.add("citation", "bibtex\n@inproceedings{...,year={2020}}")
+model_card.add("get_started_code", get_started_code).add(
+    "model_card_authors", model_card_authors
 )
+model_card.add("limitations", limitations).add("model_description", model_description)
 
-_, pkl_name = mkstemp(prefix="skops-", suffix=".pkl")
+plt.plot([1, 2, 3, 4])
+plt.ylabel("some numbers")
+plt.savefig(f"{local_repo}/fig1.png")
 
-with open(pkl_name, mode="bw") as f:
-    pickle.dump(model, file=f)
+plt.plot([4, 5, 6, 7])
+plt.ylabel("some numbers")
+plt.savefig(f"{local_repo}/fig2.png")
+
+
+model_card.add_inspection("fig1", "fig1.png")
+model_card.add_inspection("fig2", "fig2.png")
```

Comment:
should we define a section where these should go? Like, inspection, metrics, etc?

Reply:
I actually added a section previously apparently (`self.figure_paths`) but decided not to use it for reasons I don't remember 😅 I addressed this